crates/hc_sandbox/Cargo.toml

```diff
@@ -67,7 +68,10 @@ nix = { version = "0.29", features = ["process", "signal"] }
 workspace = true
 
 [features]
-default = []
+default = ["wasmer_sys"]
+
+wasmer_sys = ["holochain_client/wasmer_sys"]
+wasmer_wamr = ["holochain_client/wasmer_wamr"]
```

Comment:
This is a valid fix and I had to do it on the Rust client too.

I think the real problem though is that the way Cargo merges features in a workspace isn't reasonable. We've run into problems like this again and again where the individual crates are fine but the workspace build doesn't work. It's why I'm wanting to introduce `cargo make` so we can build the crates independently. It's also worth mentioning that problems also happen the other way round, where crates look fine in a workspace build but don't compile independently.